core/trino-spi/src/test/java/io/trino/spi/predicate/TestValueSet.java

```diff
@@ -105,6 +110,52 @@ public void testRejectRealNaNOnCreate()
                 .hasMessage("cannot use NaN as range bound");
     }
 
+    @Test
+    public void testExpand()
```

Comment:
Test should cover all the supported types, a parameterised test could be more suitable

Reply:
Good catch, done.